doc/running.rst

```diff
@@ -46,6 +46,40 @@ environment variable, such as in the following example::
    $ srun -n 512 python -m mpi4py examples/wave-eager-mpi.py'
 
 
+OpenCL device selection
+-----------------------
+
+The :mod:`pyopencl` package supports selecting the device on which to run OpenCL code.
+There are multiple ways in which this selection can happen:
```

Comment:
Clarify that it's `pyopencl.create_some_context` and [`pytest_generate_tests_for_pyopencl`](https://github.com/inducer/pyopencl/blob/310a0da934fce3d3ffa289a70e82acccbdb8783a/pyopencl/tools.py#L304-L315) that this is talking about. Also mention [`pytest_generate_tests_for_pyopencl_array_context`](https://github.com/inducer/meshmode/blob/ea35c649a9090f84512b54ddc4014025cfb1a64e/meshmode/array_context.py#L548-L583), which wrap this functionality. (Those latter two might need documentation.)

Reply:
Added.